contracts/test-utils/src/assertions.ts

```diff
@@ -42,7 +42,7 @@ async function _getInsufficientFundsErrorMessageAsync(): Promise<string> {
 }
 
 async function _getTransactionFailedErrorMessageAsync(): Promise<string> {
-    return _getGanacheOrGethErrorAsync('revert', 'always failing transaction');
+    return _getGanacheOrGethErrorAsync('revert', 'Transaction failed');
```

Comment:
The latest version of Geth changed this error message, so I had to update our assertions.